### PR TITLE
Navigation bar, hamburger menu, apis, productids, header names

### PR DESCRIPTION
**Pull request by jlatour3**
Issues #66 , #42, #40

### CODE REVIEW

File: frontend/src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  flex-direction: column;
+  height: 500px;
+`;
+
+const TextWrapper2 = styled.div`
+  display: flex;
+  border-top-style: solid;
+  align-items: center;
+  justify-content: center;
+  height: 108px;
```
Comment (Arthelon) [inline frontend/src/components/NavBar.js]: I don't think you need to modify the height attribute here. Try using padding instead to achieve the same visuals.

**Comment by jlatour3** [comment]
Added and styled headers

**Comment by jlatour3** [comment]
Issue #52

File: frontend/src/components/Header.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import styled from 'styled-components';
-import Logo from '../resources/medshare-logo.png';
+
```
Comment (Arthelon) [inline frontend/src/components/Header.js]: I think the `Header` component is no longer necessary now, instead you should directly use the `NavBar` component. The only reason we had this component was for holding the Medshare logo that was part of the old designs.

File: frontend/src/components/routerpages/Search.js [hunk 3]
```diff
@@ -2,23 +2,31 @@ import React from 'react';
...
+import ProductList from '../ProductList';
+import QueryComponents from '../QueryComponents';
+import { deviceSize } from '../../theme';
+import CategoryContainer from '../../containers/categoryContainer';
+import FeaturesContainer from '../../containers/featuresContainer';
+import SubcategoriesContainer from '../../containers/subcategoriesContainer';
+import FeatureList from '../FeatureList';
 
-const Wrapper = styled.div`
+const HeaderWrapper = styled.div`
```
Comment (Arthelon) [inline frontend/src/components/routerpages/Search.js]: I don't think you need this since the width of the parent component is already at `100%`

**Comment by Arthelon** [comment]
Also don't forget to address issue #52.

File: frontend/src/App.js [hunk 4]
```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
 import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
-import Fallback from './components/Fallback';
-import RootPage from './components/RootPage';
+import Fallback from './components/routerpages/Fallback';
```
[the remark this answers is not in the document]
Comment (jlatour3) [reply frontend/src/App.js]: done

File: frontend/src/components/Header.js (hunk 2 above)
Comment (jlatour3) [reply frontend/src/components/Header.js]: dpme

File: frontend/src/components/routerpages/Search.js (hunk 3 above)
Comment (jlatour3) [reply frontend/src/components/routerpages/Search.js]: done

File: frontend/src/components/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5rem;
+`;
+
+const TopDrawer = styled.div`
+  margin: 1rem;
+`;
+
```
[the remark this answers is not in the document]
Comment (jlatour3) [reply frontend/src/components/NavBar.js:33]: done

File: frontend/src/components/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+        <Drawer
+          style={{ background: '#6396B3', color: '#FFF' }}
+          size="255px"
+          isOpen={drawer}
+          onClose={() => handleDrawer(false)}
+          position={Position.LEFT}
+        >
+          <DrawerWrapper>
+            <TopDrawer>
+              <Button classname="bp3-minimal" icon="menu" onClick={() => handleDrawer(false)} />
```
[the remark this answers is not in the document]
Comment (jlatour3) [reply frontend/src/components/NavBar.js]: done

File: frontend/src/components/NavBar.js (hunk 1 above)
Comment (jlatour3) [reply frontend/src/components/NavBar.js]: done

**Comment by jlatour3** [comment]
finished #60, one thing to note... all the questions can open at once.. i didnt know if you wanted one card to close when another opened, and i dont really know how to start that.

**Comment by Arthelon** [comment]
Closes #42 #52 and #40

**Pull request merged**